SIGs/GRC/publications/Verifier Governance.md

```diff
@@ -0,0 +1,125 @@
+     
+2. **Availability and Performance:** A Verifier outage or inadequate performance risks bringing down its dependent services, which can be catastrophic. In case of a multi-tenant Verifier, Tenants may additionally pose performance and availability (“noisy neighbor”) threats to their peers.  
+     
+3. **Confidentiality and Integrity:** A Verifier’s failure to adequately protect its policies[^3] and secrets or generate accurate results would critically undermine the trust in all the confidential workloads dependent on it. In case of a multi-tenant Verifier, Tenants present security threats to their peers. Additionally, per “Privacy Considerations” in **\[1\]**, care should be taken to ensure confidentiality of Evidence and Attestation Results.  
+     
+4. **System of Record Considerations:** Auditors and incident responders alike will care about the Verifier Service and Verifier Tenant histories, in addition to their present state. This includes the history and governance of policies (including both Verifier Service policies and policies related to the administration of the Verifier Tenant), and the history of Attestation Results.
+
+# Solution
+
+The terms MUST/SHOULD/MAY etc. below are used in accordance with **\[2\]**. Every SHOULD recommendation is explained separately in the “SHOULD vs. MUST Clarifications” section towards the end of this document.
+
+![Solution diagram](./images/verifier_sol.png)
+
+Verifier governance aims to achieve the following goals and ensure that the evidence is in place to prove it:
+
+1. Trust in Verifier is established via the following mechanisms:  
+   1. The Verifier Service MUST be properly administered: it runs expected code in a secure configuration on hardware properly administered by the VHO.  
+   2. The Verifier Tenant MUST be executing the most current policies of that Tenant.  
+   3. An Attester SHOULD **\[a\]** know which Verifier Tenant to contact and can authenticate it before sharing any data.  
+        
+2. High availability (i.e. in excess of the availability required of the applications which depend on the Verifier) SHOULD **\[b\]** be maintained, through well-understood mechanisms:  
+   1. Maintaining multiple concurrent load-balanced Verifier instances  
+   2. Replicating this load-balanced setup in separate geographic locations  
+   3. Other common-sense measures, such as careful administration and constant monitoring for problems  
+   4. Provisioning the Verifier with sufficient computing, networking and storage resources to avoid performance degradations and outages  
+   5. For multi-tenant Verifiers, imposing throttling, quotas, or other similar measures on individual Tenants to avoid noisy neighbor issues  
```

Comment:
There's an additional aspect of verifier ability to provide services - being up to date with current sets of endorsements/reference values. This doc doesn't mention the relationship between verifiers and the Endorsers / Reference Value Provider supply chain. That also has a trust aspect in only admitting trusted RVs etc.

Reply:
I agree that managing the trust relationships with suppliers of RVs should be mentioned. My preference would be to handle this as a separate Issue, with its own PR, however.

Reply:
This seems like a pretty important aspect. We've had long discussions with customers about reference values and how timely policies need to be for enforcing the latest RVs. As a middle ground to deferring to another PR I'll take a crack at including at least some RV language and then if necessary we can improve upon it in a later draft. But if this PR is going to produce a public artifact it would be good to get RV/Endorsers mentioned even if incompletely.

Reply:
@TheBankster @SimonFrost-Arm can we resolve this if the changes proposed by @dcmiddle are acceptable?

Reply:
Sure. There still should be an issue listed to version if necessary.

Reply:
Will create issue